tests/conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+
+def pytest_addoption(parser):
```

Comment:
Does this function have to be called `pytest_addoption` for some reason? Without an underscore between add and option it looks like you've misspelled adoption

Reply:
This is unfortunately pytest boilerplate stuff: https://docs.pytest.org/en/7.1.x/example/simple.html